src/Microsoft.ML.FastTree/FastTreeRanking.cs

```diff
@@ -1090,7 +1090,7 @@ private static void PermutationSort(int[] permutation, double[] scores, short[]
                     }));
             }
 
-            [DllImport("FastTreeNative", EntryPoint = "C_GetDerivatives", CallingConvention = CallingConvention.StdCall, CharSet = CharSet.Ansi)]
+            [DllImport("FastTreeNative", EntryPoint = "C_GetDerivatives", CallingConvention = CallingConvention.Cdecl, CharSet = CharSet.Ansi)]
```

Comment:
Some functions use `StdCall` and some functions use `Cdecl` for importing things from `FastTreeNative`. Do we have different calling conventions in a single DLL?

Reply:
Not any more!

---
In reply to: [239658091](https://github.com/dotnet/machinelearning/pull/1833#discussion_r239658091) [](ancestors = 239658091)